src/themes/material/assets/mat.css

```diff
@@ -584,10 +584,12 @@ nav a.sidenav-trigger i {
 
 .twitter-bg {
     background-color: var(--twitter);
+    color: var(--twitter-hover) !important;
 }
 
 .twitter-bg:hover {
     background-color: var(--twitter-hover);
+    color: var(--twitter) !important;
 }
```

Comment:
If we are adding explicit `!important` colours to the `𝕏` button we should probably do the same for other ones. If not, we will introduce inconsistency wherever someone has overriden `btn` colours.

e.g see https://www.comicsgrid.com/article/id/15400/

Alternatively, do not use the `!important` property, though I bet you are using because the changes wouldn't take effect on the base theme styling otherwise.